openbb_platform/providers/yfinance/openbb_yfinance/models/market_indices.py

```diff
@@ -58,6 +58,31 @@ def transform_query(params: Dict[str, Any]) -> YFinanceMarketIndicesQueryParams:
         if params.get("end_date") is None:
             transformed_params["end_date"] = now
 
+        tickers = (
+            params.get("symbol").lower().split(",")
+            if "," in params["symbol"]
+            else [params.get("symbol")]
+        )
+
+        new_tickers = []
+        for ticker in tickers:
+            _ticker = ""
+            indices = pd.DataFrame(INDICES).transpose().reset_index()
+            indices.columns = ["code", "name", "symbol"]
+
+            if ticker in indices["code"].to_list():
```

Comment:
all of these to_list() can just be `.values` to optimize the usage of pandas

Reply:
replaced